Sources/Alchemy/Routing/Router.swift

```diff
@@ -90,23 +90,25 @@ public final class Router {
     /// this will parse those out from the actual URI and set them on the `Request` before
     /// passing it to the handler closure.
     ///
-    /// - Parameter request: the request this router will handle.
-    /// - Returns: a future containing the response of a handler or a `.notFound` response if there
+    /// - Parameter request: The request this router will handle.
+    /// - Throws: Any error encoutered while handling the request.
```

Comment:
Typo, "encountered"

Reply:
FYI this is WIP, opened PR to see if linux tests work